modules/tiles/test/tileset/tileset-3d.spec.js

```diff
@@ -267,6 +241,7 @@ test('Tileset3D#hasExtension returns true if the tileset JSON file uses the spec
 });
 
 test('Tileset3D#one viewport traversal', async (t) => {
+  t.plan(1);
```

Comment:
Ideally we should be able to await something `await new Promise(resolve => tileset.onTilesetLoaded(resolve))`

Reply:
We need to await a tile loading but I see the idea